pkg/users/delete-group-dialog.js

```diff
@@ -54,7 +54,12 @@ export function delete_group_dialog(group) {
                         prog.push("-f");
                     prog.push(group.name);
 
-                    return cockpit.spawn(prog, { superuser: "require", err: "message" });
+                    return cockpit.spawn(prog, { superuser: "require", err: "message" })
+                            .then(() => {
+                                if (redirect)
+                                    return cockpit.location.go("/");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230119-104710-24ec9d00-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)